### PR TITLE
Feature/tag

### PR DESCRIPTION
**Pull request by Scottjrw**
Integrated tags with backend in book history

### CODE REVIEW

File: app/history/history.html [hunk 1]
```diff
@@ -63,23 +65,27 @@ <h2>Comments</h2>
...
-        <md-option value="1">1</md-option>
-        <md-option value="0">0</md-option>
-      </md-select>
-    </md-input-container>
+    <div layout="row" >
+      <md-input-container max-width="30%">
+        <label>Ratings (0-5)</label>
+        <md-select ng-model="rating" required="">
+          <md-option value="5">5</md-option>
+          <md-option value="4">4</md-option>
```
Comment (suiyoubi) [inline app/history/history.html:73]: could use `ng-repeat` but no need to change it :)

**Comment by suiyoubi** [comment]
pull master and change line13 to `<md-icon md-font-icon="material-icons">{{item.icon}}</md-icon>{{item.label}}`

File: app/history/history.html (hunk 1 above)
Comment (Scottjrw) [reply app/history/history.html:73]: okok

**Comment by Scottjrw** [comment]
updated navigation bar

**Pull request merged**